core/histogram.cpp

```diff
@@ -108,7 +108,8 @@ DataArray histogram(const DataConstProxy &sparse,
           return transform_subspan<
               std::tuple<args<double, double, double, double>,
                          args<double, float, double, double>,
-                         args<double, float, double, float>>>(
+                         args<double, float, double, float>,
```

Comment:
This already largely existed, but where the `tuple` is defined, it would be useful to see what the individual type arguments relate to. Perhaps some king of general comment could be provided describing what the four inner types relate to regarding the histogramming?

Reply:
The template arguments to `args` have explicit names, which ones are not clear?

Reply:
Ah ok. That'll do.